c-lightning/2021-08-09-developer-call.md

```diff
@@ -0,0 +1,195 @@
+transcript_by: Michael Folkson
+categories: ['meetup']
+tags: ['lightning', 'c-lightning']
+---
+
+Name: c-lightning developer call
+
+Topic: Various topics
+
+Location: Jitsi online
+
+Date: August 9th 2021
+
+Video: No video posted online
+
+The conversation has been anonymized by default to protect the identities of the participants. Those who have expressed a preference for their comments to be attributed are attributed. If you were a participant and would like your comments to be attributed please get in touch.
+
+# Individual updates
+
+Today I started to review some pull requests. I also finished my first draft of the paid order of the metrics plugin. I have a paid order make a chart. On [lnprototest](https://github.com/rustyrussell/lnprototest) I think we can remove the Postgres dependence but I haven’t tested it and I need to work around it.
+
+That would be good, it is an indirect dependency so it would be good to get rid of it.
+
+On Raspiblitz there will be a release candidate which will include a c-lightning option to be able to run and test everything in more detail. Others have been doing work at the BTC Citadel in Germany which was good, that’s what I heard. We’ve been doing work with Tor, the censorship resistance aspect of the node which will include a menu and some scripts to aid using bridges, vanguards and authentications. This might be handy especially in view of what is going on in the regulatory side in the US now. The plan is coming together, we are starting to test and we hope we will releasing something usable soon.
+
+Someone opened a zero fee mega channel to me, I was very happy with that. There is already a movement for zero fee routing.
+
+Is that zero base fee or zero total fee?
+
+Both. Zero, zero.
+
+Is that good?
+
+My node is operating on that for months now, even before there was any movement.
+
+A race to bottom, I don’t know if it is good.
+
+It helps with channel rebalancing, if you have many peers who are zero, zero. You can rebalance for nothing. I only have one zero fee channel. Of course my channels are routing zero fee but it is one way. The other way they have some fees.
+
+Rene (Pickhardt) was excited for a while before he got excited about the routing stuff, having this local gossip, you would offer discounts to people in a localized way and say “I wouldn’t mind rebalancing so if you are up for it I will give you a freebie through this channel this way.” If you get enough of them you could rebalance, if everybody agreed. He never quite sketched out exactly how that would work. It was a cool idea. How would you identify the particular payment that was going through. There is a whole heap of questions. I always thought it was a cool protocol, you could just opportunistically gossip if you are kind of unbalanced, not ridiculously but a little bit, give people a chance. Rather than blasting out the whole network all the time that your fees are going up and down, you could do this local thing.
```

Comment:
```suggestion
Rene (Pickhardt) was excited for a while before he got excited about the routing stuff, having this local gossip, you would offer discounts to people in a localized way and say “I wouldn’t mind rebalancing so if you are up for it I will give you a freebie through this channel this way.” If you get enough of them you could rebalance, if everybody agreed. He never quite sketched out exactly how that would work. It was a cool idea. How would you identify the particular payment that was going through? There is a whole heap of questions. I always thought it was a cool protocol, you could just opportunistically gossip if you are kind of unbalanced, not ridiculously but a little bit, give people a chance. Rather than blasting out the whole network all the time that your fees are going up and down, you could do this local thing.
```

Reply:
Done